functions/helper_lib.sh

```diff
@@ -149,3 +149,25 @@ yell "# ------------------------------------------------------------------------
 # Based on the CIS Docker Benchmark 1.3.1.
 # --------------------------------------------------------------------------------------------"
 }
+
+# retrieve command for package manager
+get_list_cmd() {
+  distro="${1}"
+
+  case "$distro" in
+    fedora,rhel)
```

Comment:
Add centos?

Reply:
rocky as well?

Reply:
Added both. Please check if it's correct. Wasn't sure.